verilog/analysis/verilog_linter.cc

```diff
@@ -210,6 +211,26 @@ LinterConfiguration LinterConfigurationFromFlags() {
   const auto& ruleset = absl::GetFlag(FLAGS_ruleset);
   config.UseRuleSet(ruleset);
 
+  // Read local configuration file
+  std::ifstream local_config(".rules.verible_lint");
+  std::vector<std::string> local_rules;
+  std::string str;
+  while (std::getline(local_config, str)) {
+    local_rules.push_back(str);
+  }
+
+  if (local_rules.size() > 0) {
+    const auto& local_rules_joined = absl::StrJoin(local_rules, ",");
```

Comment:
I think we should extend our parsing to read with newline separation in addition to comma, so that we don't have to awkwardly re-assemble everything with comma.
Also, not using something called AbslParseFlag() would probably be good to avoid confusion.
 
So I suggest to decompose AbslParseFlag() and probably add a method `RuleBundle::ParseConfiguration(absl::string_view, std::string *error)`. The implementation would be lifted from `AbslParseFlag()` and slightly extended to `absl::StrSplit()` on not only comma but also newline.

Then `AbslParseFlag()` just delegates to bundle->ParseConfiguration(...)
And here in verilog_linter,  we can just local_rules_bundle.ParseConfiguration(file_content,...)

Reply:
Sure, moved this out. Added split on newline, thanks!